docs/api_docs/python/conf.py

```diff
@@ -17,8 +17,8 @@
 # -- Project information -----------------------------------------------------
 
 project = "IREE Python API"
-copyright = "2021, IREE Authors"
-author = "IREE Authors"
+copyright = "2024, The IREE Authors"
```

Comment:
Do we really want to update this?

Reply:
The copyright year? We don't need to, but I'm using the same logic as on https://github.com/iree-org/iree/pull/16028

>  We still like to intentionally update the copyright year in this one location so the website appears fresh.

Could do any of these:

* `2021` (no change)
* `2021 - 2024`
* `2024`

Reply:
Okay, fine for me if we want to follow this logic as long as we don't touch the copyright notices in source files 😆